### PR TITLE
Drupal 10 compatibility

### PR DESCRIPTION
**Pull request by AronNovak**
#41

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,22 +14,30 @@ env:
...
+        - export PATH="$PATH:$HOME/.config/composer/vendor/bin"
+        - composer global require drupal/coder:8.3.13
+        - composer global require dealerdirect/phpcodesniffer-composer-installer
+        - phpcs --config-show installed_paths
+        - phpcs --config-set installed_paths $HOME/.config/composer/vendor/drupal/coder/coder_sniffer
+        - phpcs --standard="Drupal,DrupalPractice" -p --colors --extensions=php,module,inc,install,test,profile,theme,css .
+      php: 8.1
+      env: DRUPAL_CORE=10.2.x
+      php: 8.1
+      ENV: DRUPAL_CORE=10.2.x
```
Comment (AronNovak) [inline .travis.yml]:
```suggestion
```

File: .travis.yml [hunk 2]
```diff
@@ -14,22 +14,28 @@ env:
...
+        - phpcs --standard="Drupal,DrupalPractice" -p --colors --extensions=php,module,inc,install,test,profile,theme,css .
+      php: 8.1
+      env: DRUPAL_CORE=10.2.x
 
 stages:
   - lint
   - test
 
 before_script:
+  - composer global config --no-plugins allow-plugins.dealerdirect/phpcodesniffer-composer-installer true
```
Comment (AronNovak) [inline .travis.yml]:
```suggestion
```

**Comment by amitaibu** [comment]
Last commit is red, after that MWG

**Comment by amitaibu** [comment]
(Thanks :smile: )

**Pull request merged**